doc/smarty3-templates.md

```diff
@@ -89,7 +89,7 @@ Field parameter:
 3. Help text for the input box,
 4. if set to "required" modern browser will check that this input box is filled when submitting the form,
 5. if set to "autofocus" modern browser will put the cursur into this box once the page is loaded,
-6. if set to "email", "url" or "int" modern browser will check that the filled in value corresponds to an email address, URL or numeric.
+6. if set, it will be used for the input type, default is `text` (possible types: https://www.w3schools.com/html/html_form_input_types.asp).
```

Comment:
NO NO NO NO W3SCHOOLS LINKS EVER ON MY WATCH!

They are leechers using sometimes out-of-date info to push ads and SEO their way up on Google. Use the MDN instead: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#%3Cinput%3E_types

Reply:
.. oh .. good 2 know ..